main.py

```diff
@@ -1,7 +1,42 @@
+from django.template.defaultfilters import first
+
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama'. However, this is corrected later in the code, so it's not a critical issue.